style.css

```diff
@@ -127,3 +125,190 @@ li {
 .social-icons:hover {
   transform: scale(1.1);
 }
+
+.card-container {
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+}
```

Comment:
- [ ] Kindly consider laying out elements in this container with `grid` instead of `flex` as it is a project requirement.